windows/server_stats.ps1

```diff
@@ -80,6 +80,10 @@ $ServerStats = {
         $perf = @(getPerf)
         Start-Sleep -Seconds $CpuUtilizationTimeout
         $perf += getPerf
+
+        $TimestampRaw = $perf[0].TimeStamp_Sys100NS
+        $CPUUtilizationTimestamp = [DateTime]::FromFileTimeUtc($TimestampRaw).ToString("yyyy-MM-dd hh:mm:ss")
```

Comment:
Great that this is already UTC. Not a concern of this PR, but do we also have MS4U timestamp in UTC?